portfolio/src/main/webapp/script.js

```diff
@@ -13,11 +13,11 @@
 // limitations under the License.
 
 //used to open a new tab
-function openTab(tabName) {
-  var i;
-  var x = document.getElementsByClassName("tab");
-  for(i = 0; i < x.length; i++){
-      x[i].style.display = "none";
-  }
-  document.getElementById(tabName).style.display = "block";
-}
+// function openTab(tabName) {
```

Comment:
It's fine to keep whatever you want since it's just a playaround for now but in collaboration, we will only keep code that is not commented out.